minimint/src/net/peers.rs

```diff
@@ -601,7 +601,7 @@ mod tests {
         tracing_subscriber::fmt()
             .with_env_filter(
                 EnvFilter::try_from_default_env()
-                    .unwrap_or_else(|_| EnvFilter::new("info,tide=error,minimint::net=trace")),
+                    .unwrap_or_else(|_| EnvFilter::new("info,axum=error,minimint::net=trace")),
```

Comment:
same here

Reply:
I think you can remove the `axum=error` because this tide is no longer a part of the fedimint client / server communication which this test is testing. It is only used in client <-> gateway communication.